drivers/sensor/mcp9808/Kconfig

```diff
@@ -30,6 +30,24 @@ config MCP9808_TRIGGER_OWN_THREAD
 
 endchoice
 
+choice
+	prompt "MCP9808 temperature sensor relolution"
+	default MCP9808_MEAS_RES_0_0625C
+
+config MCP9808_MEAS_RES_0_5C
+	bool "0.5C temperature resolution"
+
+config MCP9808_MEAS_RES_0_25C
+	bool "0.25C temperature resolution"
+
+config MCP9808_MEAS_RES_0_125C
+	bool "0.125C temperature resolution"
+
+config MCP9808_MEAS_RES_0_0625C
+	bool "0.0625C temperature resolution"
+
+endchoice
```

Comment:
We have been migrating toward defining these as dts properties instead of Kconfigs, as this allows different selections per instance.

Reply:
Is there an example of this being used in one of the sample projects? I'm struggling to find an example of this implementation to work from.

Reply:
See #30317